lib/nerves_runtime/log_tailer.ex

```diff
@@ -39,38 +49,36 @@ defmodule Nerves.Runtime.LogTailer do
     {:noreply, %{state | buffer: ""}}
   end
 
-  defp port_binary_name(:syslog), do: "log_tailer"
-  defp port_binary_name(:kmsg), do: "kmsg_tailer"
-
-  defp open_port(name) do
-    Port.open({:spawn_executable, executable(name)}, [
+  defp open_port(type) do
+    Port.open({:spawn_executable, executable()}, [
+      {:args, [to_string(type)]},
       {:line, 1024},
       :use_stdio,
       :binary,
       :exit_status
     ])
   end
 
-  defp executable(name) do
+  defp executable() do
     :nerves_runtime
     |> :code.priv_dir()
-    |> Path.join(name)
+    |> Path.join(@port_binary_name)
   end
 
   defp handle_message(type, data) do
     case parse_syslog_message(data) do
       %{facility: facility, severity: severity, message: message} ->
-        Logger.bare_log(
+        Logger.log(
```

Comment:
For me, the reasoning is that this is how `Logger` works normally, so it's the path of least-surprise in my mind. It's not like I'm _chooing_ to add those pieces of metadata, it's just how `Logger` is designed to work unless you explicitly turn it off. The same is true for logs generated by other applications. I imagine the purpose is so that you could analyze your logs and see which files/lines are generating errors, or filter out logs originating from a particularly chatty portion of the code that you don't really care about.

I'm fine to go back to only setting the `module` attribute, though.